### PR TITLE
HackerNews: fixed comment link, use moment.js for time

### PR DESCRIPTION
**Pull request by mattr555**
Just to start off for #1554. Also fixed a problem with the comment link that would use the javascript id of the object, rather than the actual link id.

### CODE REVIEW

File: share/spice/hacker_news/hacker_news.js [hunk 1]
```diff
@@ -12,80 +12,55 @@
...
+                normalize: function(item) {
+                    return {
+                        title: item.title,
+                        url: (item.url) ? item.url : 'https://news.ycombinator.com/item?id=' + item.objectID,
+                        points: item.points || 0,
+                        num_comments: item.num_comments || 0,
+                        post_domain: extractDomain(item.url),
+                        date_from: moment(item.created_at_i * 1000).fromNow(),
+                        arrowUrl: DDG.get_asset_path('hacker_news','arrow_up.png'),
+                        id: item.objectID
```
Comment (moollaza) [inline share/spice/hacker_news/hacker_news.js]: Looks like we don't need this anymore?

Comment (moollaza) [inline share/spice/hacker_news/hacker_news.js]: Actually, it seems like referencing `objectID` in the template is grabbing some other value...the links to the comments have much larger numbers with dashes in them and don't work for me...

**Comment by moollaza** [comment]
@mattr555 great fix! Everything looks great except the comment link -- something funny is happening with our `objectID` value...

File: share/spice/hacker_news/hacker_news.js (hunk 1 above)
Comment (moollaza) [inline share/spice/hacker_news/hacker_news.js]: nevermind, it's all good :+1:

**Comment by mattr555** [comment]
Good to go? :shipit:

**Comment by moollaza** [comment]
LGTM :+1: 

Thanks a lot @mattr555!

**Pull request merged**